tc-package.sh

```diff
@@ -4,13 +4,9 @@ set -xe
 
 mkdir -p /tmp/artifacts
 
-tar -C ${HOME}/DeepSpeech/tf/bazel-bin/tensorflow/ \
```

Comment:
I have to admit I liked it a bit better with just libtensorflow.so + libdeepspeech.so: one would be able to replace one piece without having to rebuild all.